File: Java/25. Abundant Number.java
// Sum of proper divisor is larger than the number itself
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the number to find Abundant:");
        int Abundant = userInput.nextInt();
        int add = 0;
        for (int i=1; i<Abundant; i++) {
            if(Abundant % i == 0) {
                add += i;
            }
        }
        System.out.println((add > Abundant) ? (Abundant + " is Abundant number") : (Abundant + " is not Abundant number"));
        userInput.close();
    }
}

File: Python/Important Fundamentals/25. Abundant Number.py
number = int(input("Enter the number to check Abundant:"))
sum = 0

for i in range(1,number):
    if(number % i == 0):
        sum += i
    
print(f"{number} is an Abundant Number" if(sum > number) else f"{number} is not a Abundant number")